### DISCUSSION TITLE
Different pins for touch and display. Is it possible?

### DISCUSSION THREAD

**Discussion by adam-ht** [Bodmer/TFT_eSPI#3073] [General]
Hi there, I have a question: Is there any way to use this library for display and touch when they are connected to different pins? I have display with ILI9341 display driver and with XPT2046 touch driver. The board number is esp32-24320S028R.
Display connection:
TFT_MISO 12
TFT_MOSI 13
TFT_SCLK 14
TFT_CS 15
TFT_DC 2
TFT_RST -1
TFT_BL 21

Touch connection:
TOUCH_CS 33
TOUCH_MOSI 32
TOUCH_MISO 39
TOUCH_SCLK 25
TOUCH_IRQ 36

**Comment by Bodmer** [comment]
The library does not support that, however you can use a support library such as this one which had examples for TFT_eSPI:
https://github.com/achillhasler/TFT_eTouch

**Reply by adam-ht** [reply]
Thank you for your reply.
I have already gone the easy way by just soldering some 1K ohm resistors, it works great but it would be nice to just define the touch pins in User_settings.
